### PR TITLE
Webhooks - Add more info in github actions payload example

### PR DESCRIPTION
**Pull request by ngrayluna**
## Description
Webhooks - Add more info in github actions payload example

## Ticket

https://wandb.atlassian.net/browse/DOCS-813?atlOrigin=eyJpIjoiZmIyMzk0MGQwZWJiNGUyYWFiZDQxNmM1OWQ2MWFlNGQiLCJwIjoiaiJ9

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Comment by ngrayluna** [comment]
Preview:

Screen Shot 2023-11-28 at 2 11 33 PM

**Comment by ngrayluna** [comment]
@noaleetz V1 draft created. 

Note: I'll duplicate the changes to the Artifacts Automation page when we finalize the wording on this (Model Reg -> Automation) page.

File: docs/guides/model_registry/automation.md [hunk 1]
```diff
@@ -170,6 +170,9 @@ The following tabs demonstrate example payloads based on common use cases. Withi
   }
 
   ```
+:::note
+The `event_type` key in the automation payload must match the `types` field in the GitHub workflow YAML file.
+:::
 
   Where template strings render depending on the event or model version the automation is configured for. `${event_type}` will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:
```
Comment (noaleetz) [inline docs/guides/model_registry/automation.md]:
```suggestion
  Where template strings render depends on the event or model version the automation is configured for. `${event_type}` will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:
```
@ngrayluna for some reason this sentence is troubling me - should it be depends?

File: docs/guides/model_registry/automation.md [hunk 2]
```diff
@@ -170,6 +170,9 @@ The following tabs demonstrate example payloads based on common use cases. Withi
   }
 
   ```
+:::note
+The `event_type` key in the automation payload must match the `types` field in the GitHub workflow YAML file.
```
Comment (noaleetz) [inline docs/guides/model_registry/automation.md]:
```suggestion
The `event_type` key in the webhook payload must match the `types` field in the GitHub workflow YAML file.
```

**Review by noaleetz — CHANGES_REQUESTED** [review]
left a few nits - will also ask Ken to review.

**Comment by ngrayluna** [comment]
@noaleetz Incorporated your feedback.

File: docs/guides/model_registry/automation.md [hunk 3]
```diff
@@ -170,8 +170,11 @@ The following tabs demonstrate example payloads based on common use cases. Withi
   }
 
   ```
+:::note
+The `event_type` key in the webhook payload must match the `types` field in the GitHub workflow YAML file.
+:::
 
-  Where template strings render depending on the event or model version the automation is configured for. `${event_type}` will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:
+  Where template strings render depend on the event or model version the automation is configured for. `${event_type}` will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:
```
Comment (noaleetz) [inline docs/guides/model_registry/automation.md]:
```suggestion
The contents and positioning of the rendered template strings depends on the event or model version the automation is configured for. ${event_type} will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:
```

**Review by noaleetz — CHANGES_REQUESTED** [review]
Can we add whatever changes are reflected here also inside the artifact / project level webhooks doc? https://docs.wandb.ai/guides/artifacts/project-scoped-automations#create-a-webhook-automation

**Comment by kenleejr** [comment]
Looks good to me!

**Review by kenleejr — APPROVED** [review]
Looks good to me!

**Review by noaleetz — APPROVED** [review]
lgtmmmmmmmmm

**Pull request merged**